### PR TITLE
added apidock

### PR DESCRIPTION
**Pull request by piotr-major-rmtcfm-com**

### CODE REVIEW

File: apidoc.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "name": "Express Todo API",
     "version": "1.0.0",
     "description": "Documentation for Simple Todo API with Express.js. App is using JWT, which provides authorization token. If user has logged, token is generated and is necessary to any other actions handle with VerifyToken middleware",
-    "template":{
+    "url": "127.0.0.1:3000",
```
Comment (daniel-pupczyk-rmtcfm-com) [inline apidoc.json]: we can not keep it in repo, on server it will be different

File: public/swagger/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```
Comment (daniel-pupczyk-rmtcfm-com) [inline public/swagger/index.html]: add to gitignore

File: app/docs/Auth/LoginController.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ *  @apiName PostAuthLogin
+ *  @apiGroup Auth
+ *  @apiVersion 1.0.0
+ *
+ *  @apiParam {String} email Users email
+ *  @apiParam {String} password Users password
+ *
+ *  @apiParamExample {json} Request-Example:
+ *     {
+ *       "authToken": ""
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/docs/Auth/LoginController.js]: please format it, sometimes it is 2-spaces nested, sometimes 4-spaces, in all files

File: app/docs/Auth/LoginController.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ *       "email": "test@test.com"
+ *       "password": "123456"
+ *     }
+ *
+ *  @apiSuccessExample Success-Response:
+ *     HTTP/1.1 200 OK
+ *    {
+ *        "token": TOKEN GENERATED BY JWT,
+ *        "user": {
+ *              "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/docs/Auth/LoginController.js]: format also these kind of wrong formatting

File: app/docs/Auth/ChangePasswordController.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ *    @apiParam {String} token Token which was send to users email, when asking for password reset
+ *
+ *    @apiParamExample {json} Request-Example:
+ *        {
+ *            "password": "12345678",
+ *            "password_confirmation": "12345678"
+ *        }
+ *
+ *    @apiError (400) BadRequest              Reset password token was not found </br>
```
Comment (daniel-pupczyk-prpl) [inline app/docs/Auth/ChangePasswordController.js]: `<br>` or `<br />`

File: apidoc.json (hunk 1 above)
Comment (daniel-pupczyk-prpl) [reply apidoc.json]: what about this?

File: .gitignore [hunk 6]
```diff
@@ -9,5 +9,8 @@ yarn-error.log
 package-lock.json
 apidoc
 yarn.lock
-docs
 .env
+public/*
+swagger.json
+apidoc/*
```
Comment (daniel-pupczyk-rmtcfm-com) [inline .gitignore]: why are there these two folders?

File: app/index.js [hunk 7]
```diff
@@ -40,7 +41,11 @@ app.use(function (err, req, res, next) {
     if (err.message !== 'Not allowed by CORS') return next();
     res.send({ code: 200, message: 'Request not allowed by CORS' });
 });
-app.use(helmet());
+app.use(
+    helmet({
+        contentSecurityPolicy: false
```
Comment (daniel-pupczyk-rmtcfm-com) [inline app/index.js:46]: not sure if we can change it for whole app, @rafal-ksiazek-rmtcfm-com ?

**Pull request merged**